### DISCUSSION TITLE
Mitsubishi Electric and tasmota

### DISCUSSION THREAD

**Discussion by host-felice** [arendst/Tasmota#14375] [Q&A] [answered]
Hi,
I am going to buy some Mitsubishi electric AC units and I would like to add some wemos D1 on them, but I have a question: since my wifi is getting pretty busy with those nice esp modules, is it possible to connect more than 1 unit to a wemos D1?

I know they connect with TX and RX, but tasmota can manage software TX - RX, so e.g. one AC unit connected to tx-rx and another AC unit connected to D1 and D2 as software TX-RX. GND will be in common of course.

Thanks

**Comment by sfromis** [comment, marked as answer]
AFAIK, you can't selectively switch between two serial connections with Tasmota and ESP8266.
Unless you have a really crapy wifi router/AP, the very low load from Tasmota based devices is nothing to worry about.

**Comment by host-felice** [comment]
Thanks @sfromis for your reply! That makes sense. Well, I am not that concerned about wifi load, more about the number of them :-) I love to optimize, so if I could operate 2 AC with just one wemos D1, why not? Some of them are in the opposite sides of the same wall, so... At least I can use the same level shifter for 2 AC, the one I have have 4 bidirectional ports.

**Reply by sfromis** [reply]
The number of them was part of what I meant with "Unless you have a really crappy wifi router/AP". Especially some low end ISP-supplied routers may have a limited number of possible connected devices.

**Comment by host-felice** [comment]
I have a solid Fritzbox mesh wifi, but I always try to reduce the amount of esp if I can use one esp for multiple things. Thanks for the info about multiple tx-rx!
